src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Mapping/ReviewTransferNoteViewModelMapTests.cs

```diff
@@ -107,5 +107,25 @@ public void Map_GivenValidEvidenceNoteData_ShouldContainHintItems()
             result.HintItems.ElementAt(2).Key.Should().Be("Return evidence note transfer");
             result.HintItems.ElementAt(2).Value.Should().Be("Return an evidence note transfer if there are some minor updates");
         }
+
+        [Fact]
+        public void Map_GivenAMappedTransferNoteViewModel_MustReturnMappedViewModel()
+        {
+            //arrange
+            var transfer = new ViewTransferNoteViewModelMapTransfer(TestFixture.Create<Guid>(), note, null);
+
+            var transferNoteViewModel = TestFixture.Create<ViewTransferNoteViewModel>();
+            A.CallTo(() => mapper.Map<ViewTransferNoteViewModel>(A<ViewTransferNoteViewModelMapTransfer>._)).Returns(transferNoteViewModel);
+
+            //act
+            var result = map.Map(transfer);
+
+            // assert 
+            result.ViewTransferNoteViewModel.ApprovedDate.Should().Be(transferNoteViewModel.ApprovedDate);
```

Comment:
On the asserts for the date can we add ToDisplayGMTDateTimeString to ensure the format is correct